app/views/admin/consultations/_form.html.erb

```diff
@@ -6,36 +6,35 @@
 
 <%= standard_edition_form(edition, @information, preview_design_system: true) do |form| %>
   <%= render "govuk_publishing_components/components/fieldset", {
-    legend_text: "Held on another website",
-    heading_level: 3,
-    heading_size: "l"
+    legend_text: "",
   } do %>
     <%= render "govuk_publishing_components/components/checkboxes", {
       name: "edition[external]",
       id: "edition_external",
```

Comment:
just drop this ID as we don't need it for anything so the JS won't have any effect

Reply:
👍 Good point - removed.